packages/devtools/lib/src/performance/performance.css

```diff
@@ -0,0 +1,12 @@
+.instruction-message {
+    justify-content: center;
+}
+
+.recording-message {
+    margin-top: -50px;
```

Comment:
a negative margin-top seems like a code smell. what are you trying to do that requires this?

Reply:
The negative margins are to _truly_ center the object in the container. The `center-in-parent` css class centers to the top edge of your object. But to center it vertically we need to set a negative top margin of half the container height. I wrote a comment about this in the css class.